chapter2/sxg/ex_2_5.pl

```diff
@@ -0,0 +1,7 @@
+#!perl
+print "Enter string: \n";
+$str = <STDIN>;
+print "Enter a number of times: ";
+chomp($num = <STDIN>);
+$result = $str x $num;
+print "The result is:\n$result";
```

Comment:
Yeah, my bad I forgot the result included newlines. I had forgotten the question and was expecting: 

Enter string: abc
Enter a number of times: 5
The result is: abcabcabcabcabc